client/src/pages/About/About.jsx

```diff
@@ -1,12 +1,71 @@
 import React from 'react'
 import './About.css'
+import flowers3 from '../../assets/Flowers3.png'
+import flowers2 from '../../assets/Flowers2.png'
+import treesBig from '../../assets/TreesBig.png'
+import treesSmall from '../../assets/TreesSmall.png'
+import ladyLaptop from '../../assets/LadyLaptop.png'
+import manLaptop from '../../assets/ManLaptop.png'
```

Comment:
Can we put these all in a subdirectory? eg, `assets/about/Flowers3`, and so on.

Reply:
Updated images to about_assets folder under assets